aptos-move/move-examples/dao/nft_dao/sources/nft_dao_events.move

```diff
@@ -0,0 +1,96 @@
+module dao_platform::nft_dao_events {
```

Comment:
This is not emitting the other events (or to be exact, not exposing those as functions that are then called in nft_dao module). Can you add a TODO here to do so if you're planning to do this in a follow up PR?

Reply:
I created a linear to track the TODOs including adding events.